main.cpp

```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: mana
+//  action 1: check amount of mana
+mana.checkAmount();
+//  action 2: use mana
+mana.useMana();
+//  action 3: refill mana
+mana.refillMana();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: account
+//  action 1: check balance
+account.checkBalance();
+//  action 2: debit funds
+account.debitFunds();
+//  action 3: credit funds
+account.creditFunds();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: distance
+//  action 1: total distance
+distance.totalDistanceTravelled();
+//  action 2: distance remaining
+distance.distanceLeftToGo();
+//  action 3: convert distance to km
+distance.convertToKm();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: list
+//  action 1: add item
+list.addItem();
+//  action 2: remove item
+list.removeItem();
+//  action 3: increase item count
+list.multiplyItem();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: player's weapon
+//  action 1: load weapon
+player.loadWeapon();
+//  action 2: fire weapon
+player.fireWeapon();
+//  action 3: change weapon
+player.changeWeapon();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: steeringWheel
+//  action 1: turn left
+steeringWheel.turnLeft();
+//  action 2: turn right
+steeringWheel.turnRight();
+//  action 3: align centre
+steeringWheel.alignCentre();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: magic
+//  action 1: cast spell
+magic.castSpell();
+//  action 2: choose spell
+magic.selectSpell();
+//  action 3: recharge spell
+magic.rechargeSpell();

```

Comment:
This noun doesn't perform these actions.
Something else (a different noun) performs these actions TO or WITH this noun.
- [ ] revise